README.md

```diff
@@ -4,7 +4,8 @@
 
 Rio is a MicroPaaS that can be layered on top of any standard Kubernetes cluster. Consisting of a few Kubernetes custom resources and a CLI to enhance the user experience, users can easily deploy services to Kubernetes and automatically get continuous delivery, DNS, HTTPS, routing, monitoring, autoscaling, canary deployments, git-triggered builds, and much more. All it takes to get going is an existing Kubernetes cluster and the rio CLI.
 
-Rio is currently in Alpha.
+Rio is currently in Alpha. 
```

Comment:
Break this onto a new line, in markdown only 2 lines break.

Reply:
I actually sort of like the single line view, do you think it looks better on 2 lines?

Reply:
Ehhh, yes, but either is fine.